frontend/src/framework/services/notification-service/types/notification.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type NotificationStatus } from '../enums/notification-status.enums.js';
+
+interface NotificationArguments {
```

Comment:
let's avoid interfaces and use types instead. Storing the type definition here might be unnecessary, I think it's be better to colocate it with the actual implementation.